src/commands/config.js

```diff
@@ -0,0 +1,43 @@
+const fs = require("fs");
+const os = require("os");
+const promptAsync = require("../util/promptAsync");
+
+const hiddenMaestroDirPath = () => {
+  const homedir = os.homedir();
+  const dir = "/.maestro";
+
+  return homedir + dir;
+};
+
+const createHiddenMaestroDir = () => {
+  const hiddenDirPath = hiddenMaestroDirPath();
+
+  if (!fs.existsSync(hiddenDirPath)) {
+    fs.mkdirSync(hiddenDirPath);
+  }
+};
+
+const asyncPromptForAccountInfo = async () => {
+  const accountNum = await promptAsync(
+    "Please enter your AWS Account Number: "
+  );
+  const region = await promptAsync(
+    "Please enter the region for you AWS servises (e.g. us-west-2): "
+  );
+
+  return { accountNum, region };
+};
```

Comment:
This will write to the config file `{"accountNum":"xxxxxxxxxxxx","region":"xxxxxxxxx"}`, which is a breaking change. 

Currently, the `src/aws/step-function/generateStateMachineParams` function/file uses `account_number` as the key, not `accountNum`. It also uses the `~/.config/maestro/aws_account_info.json` path, rather than ~/.maestro/aws_account_info.json`.

Either this should change from `accountNum` to `account_number`, or the `src/aws/step-function/generateStateMachineParams` should change from `account_number` to `region`.

Reply:
I vote changing `generateStateMachineParams` to `accountNum` or similar so that it is camelCase and consistent with the rest of the code base.

Reply:
Note that in either case, `src/aws/step-function/genereateStateMachineParams.js` should be reading from `~/.maestro/`, not `~/.config/maestro/`.

Reply:
> I vote changing `generateStateMachineParams` to `accountNum` or similar so that it is camelCase and consistent with the rest of the code base.

Yeah, I'm leaning this way too.

Reply:
I agree. Initially the reading from the `~/.maestro` config was a different PR. I suggest removing the reference to this PR closing issues #48 and merging this request after the spelling error is corrected. We can address changing `generateStateMachineParams` in another PR.

Reply:
Spelling error corrected and pushed.

Reply:
> I agree. Initially the reading from the `~/.maestro` config was a different PR. I suggest removing the reference to this PR closing issues #48 and merging this request after the spelling error is corrected. We can address changing `generateStateMachineParams` in another PR.

I agree that it may be the responsibility of a different PR, but because this PR introduces a breaking change, it shouldn't be merged until that other one does.